test/integration/api/images.bats

```diff
@@ -56,3 +56,15 @@ function teardown() {
 	[ "${#lines[@]}" -eq 2 ]
 	[[ "${lines[1]}" == *"image-with-labels"* ]]
 }
+
+@test "docker images imagetag" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker_swarm build -t testimage:latest $TESTDATA/imagelabel
```

Comment:
I tried using `docker tag` here but for some reason I'm seeing (what looks like) responses from the engine where `RepoTags` actually has tags from different repos (ex: `busybox:latest`, `testimage:latest`).

I haven't been able to reproduce this behaviour outside of the swarm test suite, so I'm not sure how that is happening.

This seems to work fine though.